EnglishTranslation-master/ITEM.tsv

```diff
@@ -10124,7 +10124,7 @@ ITEM_20160224_010123	Contains silver. This item will be removed after the server
 ITEM_20160224_010124	Intermediate Silver Chest
 ITEM_20160224_010125	Advanced Silver Chest
 ITEM_20160224_010126	Settlement Support Potion
-ITEM_20160224_010127	Increases your movement speed by 2, physical and magic attack by 50 for 30 minutes. {nl}Right-click to use.{nl}This item will be removed after the server maintenance of March 4th.
+ITEM_20160224_010127  Increases your movement speed and your physical/magic attack by 2 and 50, respectively, for 30 minutes.{nl}Right-click to use.{nl}This item will be removed after the server maintenance of March 4th.
```

Comment:
I forget where the conversation was, but I was talking to Spadow about this one.  The respectively thing I think functioned more of a separation device for "the 30 minutes" part, as the original makes it unclear about which parts of the buff are time restricted.  However, your way is much less complicated, and I think just having the comma will suffice to make the time portion apply to the whole sentence.